### ISSUE TITLE
Register external worker to ControlTower deployment

### ISSUE THREAD

**Issue by rhoughton-pivot** [EngineerBetter/control-tower#37]
I have a bosh operation file that adds a public-key for authenticating an external worker. I do not see any way to do this with ControlTower. Is there a different mechanism for external workers that I'm not seeing?

**Comment by DanielJonesEB** [comment]
I don't _think_ this is a supported feature. Correct me if I'm wrong, @crsimmons?

**Comment by crsimmons** [comment]
That is correct. There is currently no support for external workers in Control Tower. We have discussed adding support for arbitrary ops files in the past but for now we don't support them. You can read more about our reasoning [here](https://github.com/EngineerBetter/concourse-up/issues/84).
